docs/quick.md

```diff
@@ -2,40 +2,63 @@
 
 ## Prerequisites
 
-Before setting up cStor operators make sure your Kubernetes Cluster
+Before setting up cStor operators make sure your Kubernetes cluster
 meets the following prerequisites:
 
-1. You will need to have Kubernetes version 1.17 or higher.
-2. iSCSI initiator utils installed on all the worker nodes(If you are using rancher based cluster perform steps mentioned [here](troubleshooting/rancher_prerequisite.md)).
+1. Kubernetes version 1.17 or higher
+2. iSCSI initiator utils installed on all the worker nodes (If you are using a Rancher-based cluster, perform the steps mentioned [here](troubleshooting/rancher_prerequisite.md)).
 
 
 | OPERATING SYSTEM | iSCSI PACKAGE         | Commands to install iSCSI                                | Verify iSCSI Status         |
 | ---------------- | --------------------- | -------------------------------------------------------- | --------------------------- |
-| RHEL/CentOS      | iscsi-initiator-utils | <ul><li>sudo yum install iscsi-initiator-utils -y</li><li>sudo systemctl enable --now iscsid</li><li>modprobe iscsi_tcp</li><li>echo iscsi_tcp >/etc/modules-load.d/iscsi-tcp.conf</li></ul> | sudo systemctl status iscsid.service |
-| Ununtu/ Debian   | open-iscsi            |  <ul><li>sudo apt install open-iscsi</li><li>sudo systemctl enable --now iscsid</li><li>modprobe iscsi_tcp</li><li>echo iscsi_tcp >/etc/modules-load.d/iscsi-tcp.conf</li></ui>| sudo systemctl status iscsid.service |
-| RancherOS        | open-iscsi            |  <ul><li>sudo ros s enable open-iscsi</li><li>sudo ros s up open-iscsi</li></ui>| ros service list iscsi |
+| RHEL/CentOS      | iscsi-initiator-utils | <ul><li>sudo yum install iscsi-initiator-utils -y</li><li>sudo systemctl enable --now iscsid</li></ul> | sudo systemctl status iscsid.service |
+| Ubuntu/Debian   | open-iscsi            |  <ul><li>sudo apt install open-iscsi -y</li><li>sudo systemctl enable --now iscsid</li></ui>| sudo systemctl status iscsid.service |
 
+**Note: RancherOS is not a supported platform.**
```

Comment:
q_: why is this not supported? Is there an issue that we can point to?

Reply:
Can we use this issue?
https://github.com/rancher/os/issues/3017
cc: @prateekpandey14

Reply:
@niladrih -- Can't it be something related to DigitalOcean? Have we tried RancherOS on some lab setups?

Reply:
Yes. But, didn't create an issue. I'll create a cluster, and raise if it is incompatible. Removing the RancherOS not supported message.